src/main/java/frc/robot/subsystems/PivotSubsystem.java

```diff
@@ -90,7 +90,7 @@ public double getEncoderinRadians() {
   }
 
   public boolean atGoal(double goal) {
-    return (getMeasurement() > (goal-ManipulatorConstants.PIVOT_THRESHOLD)) && (getMeasurement() < (goal+ManipulatorConstants.PIVOT_THRESHOLD)); 
+    return (getMeasurement() > Math.abs(goal-ManipulatorConstants.PIVOT_THRESHOLD)); 
```

Comment:
remove outer parenthesis
```suggestion
    return getMeasurement() > Math.abs(goal - ManipulatorConstants.PIVOT_THRESHOLD); 
```